tests/components/zwave_js/test_websocket_api.py

```diff
@@ -16,3 +19,96 @@ async def test_websocket_api(hass, integration, hass_ws_client):
 
     assert result["client"]["ws_server_url"] == "ws://test:3000/zjs"
     assert result["client"]["server_version"] == "1.0.0"
+
+
+async def test_add_node(
+    hass, integration, client, hass_ws_client, nortek_thermostat_added_event
+):
+    """Test the add_node websocket command."""
+    entry = integration
+    ws_client = await hass_ws_client(hass)
+
+    with patch(
+        "zwave_js_server.model.controller.Controller.async_begin_inclusion",
```

Comment:
We should set the return value of the client `async_send_command` method which we've already patched.

https://github.com/home-assistant-libs/zwave-js-server-python/blob/3e996393d940fa9806fcc7eeb85e60c29328b000/zwave_js_server/model/controller.py#L139

Reply:
What's the proper way to do that? I tried a couple times to patch the return value of async_send_command but never could get it to work.

Reply:
Try:
```py
client.async_send_command.return_value = {"success": ...}
```

Reply:
The solution always ends up being easier than I expected 😂